package.json

```diff
@@ -55,6 +55,7 @@
         "build:prod:storybook": "LANGUAGE=en-US REACT=true BROWSERSLIST_ENV=production BABEL_ENV=development NODE_ENV=development build-storybook -c .storybook -o styleguide/storybook",
         "build:sync": "LANGUAGE=en-US BABEL_ENV=development NODE_ENV=development RSYNC=true webpack --config scripts/webpack.config.js --mode development",
         "build:variables": "cat `find src/styles/constants -name '*.scss'` > src/styles/variables.scss | node ./scripts/build-style-vars.js src/styles/variables.scss && rm src/styles/variables.scss",
+        "build:assets": "./scripts/asset-import/cli.js",
```

Comment:
does this need `node` in front?
`node ./scripts/asset-import/cli.js`

Reply:
oh this one includes a hashbang specifying node at the top `#!/usr/bin/env node`. any preference?